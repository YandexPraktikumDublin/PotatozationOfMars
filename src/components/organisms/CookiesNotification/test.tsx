import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { CookiesNotification } from '.'

describe('<CookiesNotification />', () => {
  it('should renders correct <CookiesNotification />', () => {
    const wrapper = shallow(<CookiesNotification />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
