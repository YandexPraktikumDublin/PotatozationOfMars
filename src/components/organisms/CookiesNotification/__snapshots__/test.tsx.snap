// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CookiesNotification /> should renders correct <CookiesNotification /> 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
